### PR TITLE
Components

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
This PR is raised to submit for code review in the Microvers dashboard of Mohammad Rafi Amin.

Description of the task to be completed 👎

I will be building my "Math magicians" website for more than a whole week. below are the tasks that will be done.
Project 1: Setup.
**Project 2: Components.**
Project 3: Events.
Project 4: Refactor with hooks.
Project 5: Full website.
Project 6: Tests (pair programming).
Exercise: Deploy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,195 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> 
+
+- [Live Demo Link]()
```
Comment (GzimAsani) [inline README.md]: - [x] The link is broken, kindly update it or mention that it will be added later

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (GzimAsani) [inline public/robots.txt]: - [x] All the boilerplate  that got created by automatically (texts, images, and styles) need to be removed(stuff such as on public folder etc)

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import Calculator from './Component/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
+
```
Comment (GzimAsani) [inline src/App.js]: - [x] App.js file needs to be class-based component, kindly update it

File: README.md [hunk 4]
```diff
@@ -1 +1,195 @@
...
+- Give a ⭐️ if you like this project!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+- This project is [MIT](https://github.com/V-Blaze/math-magicians/blob/development/LICENSE) licensed.
```
Comment (GzimAsani) [inline README.md]: - [x] The MIT link is broken, kindly update it

File: README.md [hunk 5]
```diff
@@ -1 +1,195 @@
...
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (GzimAsani) [inline README.md]: - [x] Add Acknowledgments, please remove the boilerplate

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @mrkamin             :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling, the calculator looks like the given wireframe   :+1: 

 ## Required Changes 

- [x] Check the comments below :arrow_double_down: 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mrkamin) [reply README.md]: @GzimAsani 
thanks for your feedback I will fix it but the README file was approved in my previous branch's PR I have not brink any changes to the file thanks once again I will consider it in the futuer

File: README.md (hunk 4 above)
Comment (mrkamin) [reply README.md]: @GzimAsani
thanks for your feedback I will fix it but the README file was approved in my previous branch's PR I have not brink any changes to the file thanks once again I will consider it in the future

File: README.md (hunk 5 above)
Comment (mrkamin) [reply README.md]: @GzimAsani
thanks for your feedback I will fix it but the README file was approved in my previous branch's PR I have not brink any changes to the file thanks once again I will consider it in the future

File: public/robots.txt (hunk 2 above)
Comment (mrkamin) [reply public/robots.txt]: Done

File: src/App.js (hunk 3 above)
Comment (mrkamin) [reply src/App.js]: Done thanks for your feedback

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @mrkamin ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Nice Design :white_check_mark: 
 * Good Job :+1: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@Alfredbis29 
thanks for your good and valuable feedback.

**Pull request merged**